src/services/jobs/StatisticsService.java

```diff
@@ -60,4 +67,54 @@ public StatisticsService() {
        return data; 
     }
     
+    public Map<String,Long> getGenderData(){
+       Session session = sessionFactory.getCurrentSession();
+       Transaction tx = session.beginTransaction();
+       Map<String,Long> data;
+       String male = "male", female = "female";
+
+       try{
+           data = new HashMap<>();
+           data.put( male, 
+                            (Long) session.createCriteria(User.class).
+                                   setProjection(Projections.rowCount()).
+                                   add(Restrictions.eq("gender", User.Gender.MALE)).
+                                   uniqueResult()
+                   );
+           
+           data.put( female, 
+                            (Long) session.createCriteria(User.class).
+                                   setProjection(Projections.rowCount()).
+                                   add(Restrictions.eq("gender", User.Gender.FEMALE)).
+                                   uniqueResult()
+                   );
```

Comment:
Same as before. Rather than performing two queries, you could perform just one, that is grouped by Gender.